feat(curriculum): add review DOM manipulation project (#54768)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-dom-manipulation-by-building-a-rock-paper-scissors-game/663d18c574456976d11b6e87.md

---
id: 663d18c574456976d11b6e87
title: Step 2
challengeType: 0
dashedName: step-2
---

# --description--

In the game, there will be multiple rounds. The first to reach three points wins the game.

In this step, you will focus on determining if the player has won the round.

Complete the `hasPlayerWonTheRound` function. This function has two parameters: `player` and `computer`. The function should return `true` if the player has won the round, and `false` if the player has lost or tied the round.

> Here is the criteria for the player winning a round:

Reviewer: ```suggestion
Here are the criteria for the player to win a round:
```
